andela/travel_tool_back
# #161094864 Email origin travel team if travel readiness 100%

#### What does this PR do?
This PR allows for the sending of email notifications to travel team members at the destination of a trip request upon 100 per cent of travel readiness

#### Description of Task to be completed?

1. Fetch travel team members at the the origin from where the user is travelling from.
2. Determine 100 % readiness status
3. Send email notification upon 100% travel readiness

#### How should this be manually tested?
 

1. Clone the repo - ```git clone https://github.com/andela/travel_tool_back.git```
2.  Enter into the project directory - cd travel_tool_back
3.  checkout the branch git checkout ```notify-team-origin-100%-readiness-161094864```
4.  Install dependencies using ```yarn install```
5.  Rollback and run database migrations with ```yarn db:rollmigrate```
6.  Start the server with ```make start```
7.  Create a travel request on the app.
8. Submit all checklist items as required by the destination one is travelling to.
9. After submitting all travel checklist items, assign yourself as a travel team member of the origin one is travelling to.
10. Since currently the cron-job runs after two hours, consider changing that time to about a minute or two just to save time.
11. One should receive an email about the travel completion bearing the requesters details.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#161094864

#### Screenshots (if appropriate)
<img width="1092" alt="screen shot 2018-10-23 at 10 03 41" src="https://user-images.githubusercontent.com/19623571/47342126-95749f80-d6ab-11e8-8d8d-8bbf1718d589.png">
#### Questions:
